integration-cli/docker_cli_exec_test.go

```diff
@@ -560,6 +561,10 @@ func (s *DockerSuite) TestExecWindowsOpenHandles(c *check.C) {
 			// The initial exec process (cmd.exe) has exited, and both sleeps are currently running
 			break
 		}
+		count++
+		if count >= 30 {
+			c.Error("too many retries")
```

Comment:
`c.Fatal` looks like it stops the entire CI run (unless I am mistaken reading the docs). While other tests do this, is that the behaviour we really want here? Personally I think the other tests should use `c.Error`. Either way, there's other `c.Error` calls in this test that should also be changed, or other tests should use `c.Error`